app/gui/src/model/module.rs

```diff
@@ -285,6 +285,13 @@ pub enum NotificationKind {
     },
     /// The metadata (e.g. some node's position) has been changed.
     MetadataChanged,
+    /// The source file was changed by the language server and the module content is reloaded.
+    Reloaded {
+        /// Content summary of the changed file.
+        summary: synchronized::ContentSummary,
+        /// The text change description due to parsing of the reloaded content.
+        change:  engine_protocol::language_server::TextEdit,
+    },
```

Comment:
> Describe more in docs, what does it mean "changed by language server". Give an example where it could happen (currently it's only VCS restore. In the future, there would be also some tools for refactoring).

Still valid.